module/VuFind/src/VuFind/ILS/Driver/Symphony.php

```diff
@@ -414,11 +414,11 @@ protected function getStatuses999Holdings($ids)
                     $this->translatePolicyID('LOCN', $result['current location']) :
                     $home_loc;
 
-                $available = (empty($curr_loc) || $curr_loc == $home_loc)
+                $available = ($curr_loc == $home_loc)
```

Comment:
These changes seem to assume that $curr_loc can never be empty, but I'm not convinced that is true.

Reply:
Again, there was missing return type for translatePolicyID I have overlooked